src/Components/main.js

```diff
@@ -15,6 +16,9 @@ class Main extends React.Component {
         return (
             <div>
                 <h1>Welcome to Weather Roasted</h1>
```

Comment:
Add some point we'll want to make sure the title here matches that of the readme...